.travis.yml

```diff
@@ -10,9 +10,17 @@ before_install:
   - "echo 'gem: --no-document' > ~/.gemrc"
   - gem update --system
 
+before_script:
```

Comment:
I see a lot of changes that aren't of this PR. You know why, i know it!

Reply:
Yes i know, is for the rename column genre